apps/summarize-vocabulary/main.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sort"
+
+	"github.com/golang/protobuf/proto"
+
+	metrics "github.com/googleapis/gnostic/metrics"
+)
+
+var schemas map[string]int
+var operationId map[string]int
+var parameters map[string]int
+var properties map[string]int
+
+func pbToCSV(vocab *metrics.Vocabulary) {
+	f4, ferror := os.Create("summarize-vocabulary.csv")
+	if ferror != nil {
+		fmt.Println(ferror)
+		f4.Close()
+		return
+	}
+	for _, s := range vocab.Schemas {
+		temp := fmt.Sprintf("%s,%s,%d\n", "schemas", s.Word, int(s.Count))
+		f4.WriteString(temp)
+	}
+	for _, s := range vocab.Properties {
+		temp := fmt.Sprintf("%s,%s,%d\n", "properties", s.Word, int(s.Count))
+		f4.WriteString(temp)
+	}
+	for _, s := range vocab.Operations {
+		temp := fmt.Sprintf("%s,%s,%d\n", "operations", s.Word, int(s.Count))
+		f4.WriteString(temp)
+	}
+	for _, s := range vocab.Parameters {
+		temp := fmt.Sprintf("%s,%s,%d\n", "parameters", s.Word, int(s.Count))
+		f4.WriteString(temp)
+	}
+	f4.Close()
+}
+
+func pbOutput(combinedVocab *metrics.Vocabulary) {
```

Comment:
"combinedVocab" could just be "v" (see more thoughts about variable names below)